generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -2679,7 +2679,7 @@ def read_power_scalar_f64(self, task, timeout):
         return response.voltage, response.current
 
     def read_raw(self, task, num_samps_per_chan, timeout, read_array):
-        _validate_array_dtype(read_array, numpy.generic)
+        _validate_array_dtype(read_array, numpy.uint8)
```

Comment:
Does this seem reasonable?

Reply:
I think so...

Reply:
Doesn't seem like it; tests are failing.

```
tests\component\task\test_in_stream.py:183: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
generated\nidaqmx\task\_in_stream.py:1232: in read_into
    _, samples_read, _ = self._interpreter.read_raw(
generated\nidaqmx\_grpc_interpreter.py:2682: in read_raw
    _validate_array_dtype(read_array, numpy.uint8)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

numpy_array = array([28768, 28768, 28768, 28768, 28768, 28768, 28768, 28768, 28768,
       28768, 28768, 28768, 28768, 28768, 28768, 28768, 28768, 28768,
       28768], dtype=int16)
expected_numpy_array_dtype = <class 'numpy.uint8'>

    def _validate_array_dtype(numpy_array, expected_numpy_array_dtype):
        """Raises TypeError if array type doesn't match with expected numpy.dtype"""
        if expected_numpy_array_dtype != numpy.generic and numpy_array.dtype != expected_numpy_array_dtype:
>           raise TypeError(f"array must have data type {expected_numpy_array_dtype}")
E           TypeError: array must have data type <class 'numpy.uint8'>
```

Reply:
The metadata is reverted, the tests passing now.